app/routes/me.tsx

```diff
@@ -66,57 +74,138 @@ function YouScreen() {
   const userInfo = useUserInfo()
   const requestInfo = useRequestInfo()
   const authorizeURL = getDiscordAuthorizeURL(requestInfo.origin)
+
   return (
     <div>
-      {data.message ? <div>{data.message}</div> : null}
-      <h2>User: {user.email}</h2>
-      <div>Team: {user.team}</div>
-      <div>
-        <Form method="post" action="/me">
-          <input type="hidden" name="actionId" value="logout" />
-          <button type="submit">Logout</button>
-        </Form>
-      </div>
-      <details>
-        <summary>Change account details</summary>
-
-        <Form method="post" action="/me">
-          <input type="hidden" name="actionId" value="change details" />
-          <div>
-            <label htmlFor="firstName">First Name</label>
-            <input
-              id="firstName"
+      <Form action="/me" method="post" className="mb-64 mt-12">
+        <Grid>
+          <div className="col-span-full mb-12 lg:mb-20">
+            <div className="flex flex-col-reverse items-start justify-between lg:flex-row lg:items-center">
+              <div>
+                <H2 className="mb-2">Here’s your profile.</H2>
+                <H2 variant="secondary" as="p">
+                  Edit as you wish.
+                </H2>
+              </div>
+              <Link
+                to="/logout"
+                className="flex col-span-full items-center self-end mb-12 px-11 py-6 text-black dark:text-white border-2 border-gray-600 rounded-full space-x-4 lg:col-span-8 lg:col-start-3 lg:self-auto lg:mb-0"
+              >
+                <LogoutIcon />
+                <H6 as="span">logout</H6>
+              </Link>
+            </div>
+          </div>
+
+          <div className="col-span-full mb-24 lg:col-span-5 lg:mb-0">
+            <Label className="mb-4" htmlFor="firstName">
+              First name
+            </Label>
+
+            <Input
+              className="mb-8"
               name="firstName"
+              id="firstName"
+              autoComplete="firstName"
               defaultValue={user.firstName}
+              required
             />
+
+            <Label className="mb-4" htmlFor="email-address">
+              Email address
+            </Label>
+
+            <Input
+              name="email"
+              id="email-address"
+              autoComplete="email"
+              required
+              defaultValue={user.email}
+              className="mb-8"
+            />
```

Comment:
Should the email be editable, or `readOnly`?

Reply:
Hmmm.... Let's start with readonly. Maybe I'll change it later.